front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PluginVersionController.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.front50.controllers;
+
+import com.netflix.spinnaker.front50.model.plugins.PluginInfo;
+import com.netflix.spinnaker.front50.model.plugins.PluginVersionPinningService;
+import java.util.Map;
+import lombok.Value;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/pluginVersions")
+public class PluginVersionController {
```

Comment:
Do we need an endpoint for defining the "latest" versions for a service?  I.E., in the case wherein the server group is unknown (because it has not been deployed yet) and we want to define the versions for that next server group.

Reply:
I don't think that would be an endpoint that falls in scope of this PR. We'll need ways to actuate the versions, but there's a lot more than just that endpoint involved in making the feature work - I'd rather tackle that when we get to the overall feature. WDYT?

Reply:
That works for me. 👍